### PR TITLE
Comparison page

### PR DESCRIPTION
**Pull request by samnixon87**
First cut of the Roadie Comparison page

**Before:**

image

**After:**

image

**Full:**

comparison page

### CODE REVIEW

File: src/components/SitewideHeader/SitewideHeader.js [hunk 1]
```diff
@@ -126,19 +126,14 @@ const SitewideHeader = ({
 
           <Popover.Group as="nav" className="hidden md:flex space-x-10">
             <TopNavDropdownItem text="Product" subItems={features} />
+            <TopNavItem to="/backstage-comparison/" text="Roadie vs Backstage" />
             <TopNavItem to="/backstage-weekly/" text="Backstage Weekly" />
```
Comment (dtuite) [inline src/components/SitewideHeader/SitewideHeader.js:130]: I think we can temporarily replace this page to make more space in the header. At least until we manage to revive the newsletter.

```suggestion
```

File: src/pages/backstage-comparison/index.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { SEO, SitewideHeader, SitewideFooter, Headline, Link } from 'components';
+import { SectionHeader, FeatureComparisonTable } from 'components/BackstageComparison';
+import { FAQs } from 'components/landing';
+import RoadieBackstageLogos from '../../../content/assets/roadie-backstage-logos.svg';
+
+const Pricing = ({
```
Comment (dtuite) [inline src/pages/backstage-comparison/index.js:8]: Component should be renamed

File: src/pages/backstage-comparison/index.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    site: {
+      siteMetadata: { title: siteTitle },
+    },
+  },
+}) => {
+  return (
+    <>
+      <SEO
+        title={`Roadie vs Backstage | ${siteTitle}`}
+        description="Hosted Backstage from Roadie. Simple, flexible options to fit your IDP needs. Free Trial."
```
Comment (dtuite) [inline src/pages/backstage-comparison/index.js:19]: Description should more accurately describe the page contents.

File: src/pages/backstage-comparison/index.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        <div className="flex justify-center pb-12">
+          <img src={RoadieBackstageLogos} alt="" className="webkit-optimize-image-rendering" />
+        </div>
+        <div className="flex justify-center">
+          <div className="flex-column text-center max-w-2xl mt-5 mx-auto sm:px-6 lg:px-8">
+            <Headline el="h2" size="small">
+              Overview
+            </Headline>
+            <h2 className="text-lg sm:mt-8 lg:text-xl xl:text-xl">
+              <Link to="https://backstage.io/">Backstage.io</Link> is the best tool on the market
```
Comment (dtuite) [inline src/pages/backstage-comparison/index.js:39]: Anyone is here probably already knows what Backstage is, we can drop the `.io` in the text.

Adding `color="primary"` makes the link easier to spot.

```suggestion
              <Link to="https://backstage.io/" color="primary">Backstage</Link> is the best tool on the market
```

File: src/pages/backstage-comparison/index.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+              for building Internal Developer Portals (IDP) but requires significant investment to take
```
Comment (dtuite) [inline src/pages/backstage-comparison/index.js:40]:
```suggestion
              for building Internal Developer Portals (IDPs). However, it requires significant investment to take
```

File: src/pages/backstage-comparison/index.js [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+              the raw building blocks that Backstage offers and turn them into a true Internal
+              Developer Portal.
+            </h2>
+            <h2 className="text-lg sm:mt-8 lg:text-xl xl:text-xl">
+              With Roadie, we've taken the open-source version of Backstage and used it to build a
+              comprehensive hosted IDP for everyone to use.
```
Comment (dtuite) [inline src/pages/backstage-comparison/index.js:46]:
```suggestion
              comprehensive and maintenance-free IDP.
```

File: src/components/BackstageComparison/FeatureComparisonTable.js [hunk 7]
```diff
@@ -0,0 +1,413 @@
...
+  {
+    name: 'Software catalog',
+    features: [
+      {
+        name: 'Bring your own data model',
+        competitor: { Roadie: 'Yes', Backstage: 'Yes, with some effort' },
+      },
+      {
+        name: 'Entity definition',
+        competitor: { Roadie: 'Markdown, UI, or via API', Backstage: 'Markdown' },
```
Comment (dtuite) [inline src/components/BackstageComparison/FeatureComparisonTable.js:17]: What does "Entity definition" mean in this context? I actually don't understand.

File: src/components/BackstageComparison/FeatureComparisonTable.js [hunk 8]
```diff
@@ -0,0 +1,413 @@
...
+      },
+      {
+        name: 'API-based ingestion and update',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Out of the box data ingestion (k8s, github, CI/CD, etc)',
```
Comment (dtuite) [inline src/components/BackstageComparison/FeatureComparisonTable.js:24]: How come we're comfortable giving Backstage a `yes` and Port are not?

File: src/components/BackstageComparison/FeatureComparisonTable.js [hunk 9]
```diff
@@ -0,0 +1,413 @@
...
+  {
+    name: 'TechDocs',
+    features: [
+      {
+        name: 'Docs creation engine',
+        competitor: { Roadie: 'TechDocs', Backstage: 'TechDocs' },
+      },
+      {
+        name: 'Extensibility via plugins',
+        competitor: { Roadie: true, Backstage: true },
```
Comment (dtuite) [inline src/components/BackstageComparison/FeatureComparisonTable.js:97]: I think there's a benefit to add here about how we build docs in the background by default?

File: src/components/BackstageComparison/FeatureComparisonTable.js [hunk 10]
```diff
@@ -0,0 +1,413 @@
...
+  },
+  {
+    name: 'Other plugins and extensions',
+    features: [
+      {
+        name: 'Available Plugins',
+        competitor: { Roadie: '70+', Backstage: 'Requires build' },
+      },
+      {
+        name: 'Access to on-prem infrastructure and resources',
```
Comment (dtuite) [inline src/components/BackstageComparison/FeatureComparisonTable.js:109]:
```suggestion
        name: 'Secure access to on-prem infrastructure and resources',
```

File: src/components/BackstageComparison/FeatureComparisonTable.js [hunk 11]
```diff
@@ -0,0 +1,413 @@
...
+        name: 'Custom roles',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Custom permissions policies',
+        competitor: { Roadie: true, Backstage: false },
+      },
+    ],
+  },
+  {
```
Comment (dtuite) [inline src/components/BackstageComparison/FeatureComparisonTable.js:173]: Given the conversation we had 20 mins ago, can we add a cost insights section?

File: src/components/BackstageComparison/FeatureComparisonTable.js [hunk 12]
```diff
@@ -0,0 +1,413 @@
...
+      },
+      {
+        name: 'Support',
+        competitor: {
+          Roadie: 'Email, Slack/Teams, Discord, and 24/7 On-call',
+          Backstage: 'via the OSS community',
+        },
+      },
+      {
+        name: 'Cost',
```
Comment (dtuite) [inline src/components/BackstageComparison/FeatureComparisonTable.js:196]: What does this one mean?

File: src/components/SitewideHeader/SitewideHeader.js (hunk 1 above)
Comment (samnixon87) [reply src/components/SitewideHeader/SitewideHeader.js:130]: Ok, sure. I think we should keep it + revive it now, but I can open another PR for it to have that discussions.

File: src/components/BackstageComparison/FeatureComparisonTable.js (hunk 8 above)
Comment (samnixon87) [reply src/components/BackstageComparison/FeatureComparisonTable.js:24]: For which section?

File: src/components/BackstageComparison/FeatureComparisonTable.js (hunk 12 above)
Comment (samnixon87) [reply src/components/BackstageComparison/FeatureComparisonTable.js]: I was going to link out to a cost comparison, but feels non-trivial for the first pass. i've removed

File: src/components/BackstageComparison/FeatureComparisonTable.js (hunk 9 above)
Comment (samnixon87) [reply src/components/BackstageComparison/FeatureComparisonTable.js:93]: Not sure how we'd phrase it, so I've asked on Slack. I'm ok to leave this out this time

File: src/components/BackstageComparison/FeatureComparisonTable.js [hunk 13]
```diff
@@ -0,0 +1,426 @@
...
+  {
+    name: 'Software catalog',
+    features: [
+      {
+        name: 'Bring your own data model',
+        competitor: { Roadie: 'Yes', Backstage: 'Yes, with some effort' },
+      },
+      {
+        name: 'How you add items to the Catalog',
+        competitor: { Roadie: 'Markdown, UI, or via the Roadie API', Backstage: 'Markdown' },
```
Comment (dtuite) [inline src/components/BackstageComparison/FeatureComparisonTable.js:17]:
```suggestion
        competitor: { Roadie: 'YAML, UI, or via the Roadie API', Backstage: 'YAML' },
```

?

File: src/components/SitewideHeader/SitewideHeader.js (hunk 1 above)
Comment (dtuite) [reply src/components/SitewideHeader/SitewideHeader.js:130]: Meh. Up to you.

File: src/components/BackstageComparison/FeatureComparisonTable.js (hunk 8 above)
Comment (dtuite) [reply src/components/BackstageComparison/FeatureComparisonTable.js]: "Out of the box data ingestion"

**Review by dtuite — APPROVED** [review]

File: src/components/BackstageComparison/FeatureComparisonTable.js [hunk 14]
```diff
@@ -0,0 +1,434 @@
...
+const sections = [
+  {
+    name: 'Software catalog',
+    features: [
+      {
+        name: 'Bring your own data model',
+        competitor: { Roadie: 'Yes', Backstage: 'Yes, with some effort' },
+      },
+      {
+        name: 'Adding and updating Catalog items via yaml',
```
Comment (dtuite) [inline src/components/BackstageComparison/FeatureComparisonTable.js:16]:
```suggestion
        name: 'Adding and updating Catalog items via YAML',
```

**Pull request merged**